roles/etcd/templates/etcd-rkt.service.j2

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=etcd rkt wrapper
+Documentation=https://github.com/coreos/etcd
+Wants=network.target
+
+[Service]
+Restart=on-failure
+RestartSec=10s
+TimeoutStartSec=0
+LimitNOFILE=40000
+
+ExecStart={{ rkt_bin_dir | default("/usr/bin") }}/rkt run \
+--uuid-file-save=/var/run/etcd.uuid \
+--volume=etc-ssl-certs,kind=host,source=/etc/ssl/certs,readOnly=true \
+--mount=volume=etc-ssl-certs,target=/etc/ssl/certs \
+--volume=etcd-cert-dir,kind=host,source={{ etcd_cert_dir }},readOnly=true \
+--mount=volume=etcd-cert-dir,target={{ etcd_cert_dir }} \
+--volume=var-lib-etcd,kind=host,source=/var/lib/etcd,readOnly=false \
+--mount=volume=var-lib-etcd,target=/var/lib/etcd \
+--set-env-file=/etc/etcd.env \
+--stage1-from-dir=stage1-fly.aci \
+{{ etcd_image_repo }}:{{ etcd_image_tag }} \
+--name={{ etcd_member_name | default("etcd") }}
+
+ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/run/etcd.uuid
```

Comment:
rkt doesn't seem to care too much about the container/pod name, I can have X amount of nginx containers named nginx running and rkt is happy with that[1], so that's why I opted to use the `--uuid-file-save` arg with rkt run. 

[1]
```
UUID		APP			IMAGE NAME					STATE		CREATED		STARTED		NETWORKS

3c809c62	nginx			registry-1.docker.io/library/nginx:latest	running		19 seconds ago	12 seconds ago	
6e3228d4	nginx			registry-1.docker.io/library/nginx:latest	running		1 minute ago	30 seconds ago	default:ip4=172.16.28.7
af81e2ec	nginx			registry-1.docker.io/library/nginx:latest	running		1 minute ago	34 seconds ago	default:ip4=172.16.28.6
b555c8db	nginx			registry-1.docker.io/library/nginx:latest	running		1 minute ago	1 minute ago	default:ip4=172.16.28.5
```